src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23801.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue23801 : _IssuesUITest
+	{
+		public override string Issue => "Span GestureRecognizers don't work when the span is wrapped over two lines";
+
+		public Issue23801(TestDevice device)
+		: base(device)
+		{ }
+
+		[Test]
+		[Category(UITestCategories.Label)]
+		[FailsOnWindows]
```

Comment:
The fix looks good to me, but, do we know why the test is failing on Windows?
Could you modify the test to include a comment like:

`[FailsOnWindows("Span Gestures don't work on Windows when span is wrapped because blabla...")]`

Reply:
@jsuarezruiz, I tried to automate this case by locating the span position and tapping it. Unfortunately, it is not working on the Windows platform in CI, which is why I initially added the FailsOnWindows attribute.

I have now removed the FailsOnWindows attribute and instead added a condition to run the test case on all platforms except Windows.